lab-steven-johnson/server.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const bp = require('./lib/body-parser');
+const http = require('http');
+const url = require('url');
+const queryString = require('querystring');
+const cowsay = require('cowsay');
+const PORT = process.env.PORT || 3000;
+
+const server = module.exports = http.createServer(function(req, res){
+  req.url = url.parse(req.url);
+  req.url.query = queryString.parse(req.url.query);
+
+  if (req.method === 'POST') {
+
+    if (req.url.pathname === '/cowsay') {
+      bp(req, function(err){
+        if (err) console.error(err);
+        let message = cowsay.say({text: req.body.text});
```

Comment:
could do an -f flag here to incorporate tux instead of a cow rendering :P    if you're curious, check the docs for other animals :)